.storybook/pages/CadStep1/CadStep1.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+
+import {
+  Button,
+  ButtonGroup,
+  Heading,
+  HorizontalStepper,
+  Icon,
+  Layout,
+  LayoutContainer,
+  LayoutSection,
+  PageHeader,
+  Panel,
+  Text,
+} from '../../../src';
+import TextField from '../../../src/upcoming-components/TextField';
+
+import '../../../src/components/Utilities/Spacing.css';
+import '../../../src/components/Utilities/TypographyUsage.css';
+
+export const CadStep1 = () => (
+  <>
+    <Panel variant="squared">
+      <LayoutContainer>
+        <PageHeader
+          className="u-margin-bottom-none"
+          description={
+            <HorizontalStepper
+              activeIndex={0}
+              steps={[
+                'Add classroom details',
+                'Add projects',
+                'Create course plan',
+              ]}
+            />
+          }
+          headingSize="h5"
+          right={
+            <ButtonGroup>
+              <Button>Return to Course</Button>
+              <Button variant="primary">
+                Next <Icon name="arrow-narrow-right" purpose="decorative" />
+              </Button>
+            </ButtonGroup>
+          }
+          title="Course Planner: History 6"
+        />
+      </LayoutContainer>
+    </Panel>
+    <LayoutContainer>
+      <Layout className="u-padding-xl" variant="50-50">
```

Comment:
This doesn't work well responsively. I think we need to talk with design about how to accommodate this in more of a methodical manner. Let's remove this for now and then add a `u-margin-bottom-lg` to the `Panel` for now.

Reply:
Will do